public/link.css

```diff
@@ -245,6 +250,7 @@ a:hover > .link {
     z-index: -1;
     mix-blend-mode: color-dodge;
     opacity: 0.3;
+    filter: contrast(150%) saturate(200%);
```

Comment:
originally the color-dodge here also blended with the site background, but that's not possible when it's rendered as its own GPU surface, so the colors look a bit more washed out, especially on emojis. This isn't perfect but it makes them look acceptable (imo)